pkg/destroy/gcp/gcp.go

```diff
@@ -90,6 +92,18 @@ func (o *ClusterUninstaller) Run() error {
 		return errors.Wrap(err, "failed to create compute service")
 	}
 
+	req := o.computeSvc.MachineTypes.AggregatedList(o.ProjectID).Fields("items/*/machineTypes(name,guestCpus),nextPageToken")
```

Comment:
I imagine this will be contentious - I need a place to put this cache, open to ideas.